projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -365,19 +365,54 @@ public static function parse( $attributes, $content ) {
 			 * @param string $contact_form_id Contact form post URL.
 			 * @param $post $GLOBALS['post'] Post global variable.
 			 * @param int $id Contact Form ID.
+			 * @param int $page Page number.
 			 */
 			$url                     = apply_filters( 'grunion_contact_form_form_action', "{$url}#contact-form-{$id}", $GLOBALS['post'], $id, $page );
 			$has_submit_button_block = str_contains( $content, 'wp-block-jetpack-button' );
 			$form_classes            = 'contact-form commentsblock';
 			$post_title              = $post->post_title ?? '';
 			$form_accessible_name    = ! empty( $attributes['formTitle'] ) ? $attributes['formTitle'] : $post_title;
-			$form_aria_label         = isset( $form_accessible_name ) && ! empty( $form_accessible_name ) ? 'aria-label="' . esc_attr( $form_accessible_name ) . '"' : '';
+			$form_aria_label         = isset( $form_accessible_name ) && ! empty( $form_accessible_name ) ? esc_attr( $form_accessible_name ) : '';
 
 			if ( $has_submit_button_block ) {
 				$form_classes .= ' wp-block-jetpack-contact-form';
 			}
 
-			$r .= "<form action='" . esc_url( $url ) . "' method='post' class='" . esc_attr( $form_classes ) . "' $form_aria_label enctype='multipart/form-data' novalidate>\n";
+			/**
+			 * Filter the contact form attributes that are applied to the form element.
+			 *
+			 * @module contact-form
+			 *
+			 * @since $$next-version&&
+			 *
+			 * @param array $attributes Contact form attributes. Array of key value pairs.
+			 * @param $post $GLOBALS['post'] Post global variable.
+			 * @param int $id Contact Form ID.
+			 * @param int $page Page number.
+			 */
+			$attributes = apply_filters(
+				'jetpack_form_attributes',
+				array(
+					'aria-label' => $form_aria_label,
+					'class'      => $form_classes,
+				),
+				$GLOBALS['post'],
+				$id,
+				$page
+			);
+
+			// Remove the form enctype attribute if it was applied.
+			remove_filter( 'jetpack_form_attributes', array( 'Automattic\Jetpack\Forms\ContactForm\Contact_Form', 'add_enctype_attribute' ) );
```

Comment:
Can you clarify why we're needing this remove filter call? I'm trying to find other references to add_enctype_attribute in this file or elsewhere, but I don't see any...

Reply:
The Static Analysis test is failing related to this line as well. Error is:

```
projects/packages/forms/src/contact-form/class-contact-form.php:405
Error: UndefError PhanUndeclaredStaticMethodInCallable Reference to undeclared static method \Automattic\Jetpack\Forms\ContactForm\Contact_Form::add_enctype_attribute in callable
```

Reply:
Update: This line is removing the same filter added elsewhere in the PR, but the method name was wrong. I pushed a fix.